libs/frontend/feature-render/src/lib/get-page-style.ts

```diff
@@ -46,6 +48,34 @@ export const getBuildPageStyle = (site: ISite, page: IPage): IRawStyleRecord =>
   return pageStyle
 }
 
+export const getRootBackgroundStyle = (
```

Comment:
@abemscac This is messy, maybe there's a better way to do it. It just grabs the `background` and `background-color` properties from the page root for all breakpoints.